### ISSUE TITLE
RPC get_transaction_pool not json serializable

### ISSUE THREAD

**Issue by bianchimro** [monero-project/monero#3174]
RPC method `get_transaction_pool` in master branch (I'm looking at commit 5f09d6c) returns an object with the `tx_blob`, not present in the latest released version (0.11.1.0).

IMHO this creates two problems:

- the content of this key is not JSON serializable
- the serialization takes quite a long time, resulting in very slow response of this API call 

I see two possible options:

- adding an optional parameter to `get_transaction_pool` for omitting `tx_blob`
- create a separate endpoint for this purpose

**Comment by moneromooo-monero** [comment]
I'd be fine with a flag. There's a get_transaction_pool_hashes call too, wihch is meant to be faster (you then ask for tx data for those txes you haven't seen before). I think the blob should be the hex representation of that tx, not the raw binary anyway, and that'd be JSON friendly.

**Comment by bianchimro** [comment]
@moneromooo-monero thanks for your quick response.

Didn't know about the get_transaction_pool_hashes, thanks for pointing out.

And yes, the hex representation of the tx would be JSON serializable, as it happens in other RPC calls.

**Comment by PochenriederAlex** [comment]
I support the idea of returning tx_blob in hex representation just like gettransactions call. I used to call this method but since the tx_blob was added  i can't parse the response.

@moneromooo-monero  get_transaction_pool_hashes have also a mixed format response.
It's not a full binary response. It's a json with a Binary stream in the status field so it's not JSON serializable either.

So, there's another way i can get mempool transactions or at least the hashes to call gettransactions later?
